src/functionalTest/java/uk/gov/hmcts/reform/draftstore/data/PaginationTest.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.draftstore.data;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import uk.gov.hmcts.reform.draftstore.domain.Draft;
+import uk.gov.hmcts.reform.draftstore.endpoint.v3.helpers.SampleData;
+
+import java.util.List;
+import java.util.function.Supplier;
+
+import static java.lang.Integer.parseInt;
+import static java.util.stream.Collectors.toList;
+import static java.util.stream.IntStream.range;
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest
+@ActiveProfiles("test")
+public class PaginationTest {
+
+    @Autowired
+    private DraftStoreDAO repo;
+
+    @Test
+    public void should_limit_number_of_results() throws Exception {
```

Comment:
Not sure if checked generic Exception is necessary. Looks like could be removed.

Reply:
That's just what Intellij generated by default (I'm not sure why though :)).
Most test methods in this project have such test method signatures.

Reply:
its just in the default intellij template for creating tests.
Probably fine